src/main/java/roomescape/presentation/controller/view/AuthViewController.java

```diff
@@ -0,0 +1,12 @@
+package roomescape.presentation.controller.view;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+
+@Controller
+public class AuthViewController {
+    @GetMapping("/login")
+    public String login() {
+        return "login";
+    }
+}
```

Comment:
요 부분도 다른 컨트롤러와 통일성을 위해 RequestMapping 활용해도 좋을 것 같네요!

```suggestion
@Controller
@RequestMapping("/login")
public class AuthViewController {
    @GetMapping
    public String login() {
        return "login";
    }
}
```